packages/components/HowItWorksSection.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Icon, Box, useColorModeValue, Text, Heading } from '@chakra-ui/react';
+import { GoCode, GoBriefcase, GoCommentDiscussion } from 'react-icons/go';
+import { SimpleGrid } from '@chakra-ui/react';
+
+const HowItWorksSection = () => {
+  return (
+    <Box bg={useColorModeValue('gray.50', 'gray.900')} p="1rem" mt="3rem">
+      <SimpleGrid
+        minChildWidth="150px"
+        spacing="50px"
+        textAlign="center"
+        margin={20}
```

Comment:
também é bom manter consistencia entre medidas sempre que possível

Reply:
tambem to achando estranho isso. eu segui o padrao de codigo que vi em outros arquivos mas mesmo assim acho que seria melhor ter consistencia nestas unidades de medida tb

Reply:
todo